timewarp_gl/plugin.cpp

```diff
@@ -223,27 +185,27 @@ class timewarp_gl : public threadloop {
         return pixels;
     }
 
-    void BuildTimewarp(HMD::hmd_info_t* hmdInfo) {
+    void BuildTimewarp(HMD::hmd_info_t& hmdInfo) {
```

Comment:
Not as it is, no. Because it generates the distortion buffers, which are class members. We could refactor and pass the buffers as function parameters, but that may or may not be worth it. Also, we're moving towards Vulkan anyway now.